server/datastore/mysql/hosts_test.go

```diff
@@ -644,7 +644,7 @@ func testHostsWithTeamPackStats(t *testing.T, ds *Datastore) {
 			PackID:             0, // pack_id will be 0 for stats of queries not in packs.
 			LastFetched:        nil,
 			DiscardData:        tpQuery.DiscardData,
-			AutomationsEnabled: tpQuery.AutomationsEnabled,
+			AutomationsEnabled: false, // scheduled queries don't have automations_enabled field
```

Comment:
@lucasmrod I had to change this to get the test to pass. Scheduled query table does not have automations_enabled column. Or am missing how this test is supposed to work?

Reply:
@mostlikelee found the issue, I will fix.